Info added on Requirement for /25 Subnet and Public IP SKU of type St…

mbsnl opened the pull request:
…andard

mbsnl committed:
Info added on Requirement for /25 Subnet and Public IP SKU of type Standard

vhorne reviewed (changes requested):
Please change the notes to read:
The minimum size of the AzureFirewallSubnet subnet is /25.
The Public IP address must be the Standard SKU type.

mbsnl committed:
Update tutorial-firewall-deploy-portal.md

Suggested by vhorne: 
The minimum size of the AzureFirewallSubnet subnet is /25.
The Public IP address must be the Standard SKU type.
